requirements.txt

```
@@ -1,2 +1,4 @@
 ai2thor>=0.0.34
-scikit-image>=0.14.1
\ No newline at end of file
+scikit-image>=0.14.1
+pytorch>=0.4.1
+
```

Add also 
```suggestion
vizdoom # needed for vizdoom experiments
```
or similar

---

but we don't want to discourage anyone from using our repo. ViZDoom is extremely hard to install and has so many dependencies. Our repo should be easy to start for 95% of settings.
The second you try vizdoom it will crash.
So my solution is if comments work in .txt. fles
`#vizdoom # needed for vizdoom experiments`

---

vizdoom is as hard to install as `sudo tons of ubuntu packages`, then `pip install vizdoom`. For me it took 2 minutes in windows (not supported by us) and ubuntu 16 and 18.
If you don't want to add it to requirements then add a comment in the readme to "install vizdoom if you want to run xxx experiments. Installations can be found here (link).

---

you are one of the lucky ones

but added :

(you will need to install ViZDoom) in README.md in the vizdoom example command